### PR TITLE
Reverts transpilation of eslint config

### PR DESCRIPTION
**Pull request by jamesseanwright**
Reverts transpilation of eslint-config-thetimes, as it's overkill.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/4be0e48d49373bbe21ec8e463f08a569d8b94943/index.html

File: packages/eslint-config-thetimes/package.json [hunk 1]
```diff
@@ -2,14 +2,11 @@
   "name": "@times-components/eslint-config-thetimes",
   "version": "0.1.6",
   "description": "The eslint config for Times Components",
-  "main": "dist/index.js",
```
Comment (colincclark) [inline packages/eslint-config-thetimes/package.json]: no need for dist in the .prettierignore any more then?

Comment (jamesseanwright) [reply packages/eslint-config-thetimes/package.json]: Good spot, I'll remove this tomorrow.

Comment (jamesseanwright) [reply packages/eslint-config-thetimes/package.json]: @colincclark It looks like this was in place before my changes; someone had copied the `.prettierignore` from another package. I've nonetheless removed any redundant entries.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/5f8bbf30395518df6dc42e4ea1b34512bc22011f/index.html

**Pull request merged**